portfolio/src/main/webapp/script.js

```diff
@@ -77,3 +77,100 @@ function createListElement(text) {
   liElement.innerText = text;
   return liElement;
 }
+
+/** Places I have lived. */
+var placesLived = [
+  ['Evanston', 42.0451, -87.6877, '2000-2002', 'I was born in outside Chicago at Northwestern Medical Center.'],
+  ['Overland Park', 38.9822, -94.6708, '2002-2017', 'I grew up in the Kansas City suburbs, first in Lenexa and then Overland Park. I attended Blue Valley North high school for three years.'],
+  ['Wassenaar', 52.1429, 4.4012, '2017-2018', 'I moved to the Netherlands before my senior year because my father switched jobs. I graduated from the American School of the Hague.'],
+  ['Athens', 33.9519, -83.3576, '2018-Present', 'I now live in Athens, Georgia. I am going into my junior year at UGA.'],
+  ['Oxford', 51.7520, -1.2577, 'May-June 2019', 'For the beginning of summer after my freshman year, I studied Computing Ethics at Oxford through the Foundation Fellowship\'s Maymester Program']
+];
+
+/** Places I have visited. */
+var placesVisited = [
+  // Domestic
```

Comment:
Presumably you meant to add a comment down below when you started listing international places?